README.adoc

```diff
@@ -15,7 +15,7 @@ to `~/go`)
 cd $GOPATH   #default to ~/go
 git clone https://github.com/ericbottard/kubernetes-model src/github.com/fabric8io/kubernetes-model/
 cd src/github.com/fabric8io/kubernetes-model/
-git checkout sk8s-sidecar
+git checkout sk8s-sidecar-scaling
```

Comment:
Pay attention to this (esp if merging after https://github.com/markfisher/sk8s/pull/38)